spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1106,6 +1106,7 @@ def _run_algorithms(
             try:
                 self._shutdown()
                 self.write_finished_file()
+                self.write_errored_file()
```

Comment:
This means you are going to write it when there isn't an error?

Reply:
this is inside the except of the run_algorithm. so to even be here, its had some form of error. it may be recoverable to shutdown and "finish" but it finished via a error state. so no it'll write it when there has been an error. unless ive misread the code again

Reply:
Ah OK.  I just noticed, this should also write the finished and errored files outside of the try catch I think (this isn't strictly part of this PR. but easy enough to fix whilst here).  This would mean that it would write the files even if shutdown were to fail.